src/components/GlobalStyles/breakpoints.js

```diff
@@ -0,0 +1,28 @@
+const size = {
+  xs: '480px',
+  sm: '768px',
+  md: '992px',
+  lg: '1200px',
+  xlg: '1440px'
+}
+
+const device = {
+  maxXS: `(max-width: ${size.xs})`,
```

Comment:
Might be worth testing the max-width variables, I feel like in the past I've had to subtract 1 from these so the max-width and the min-width values didn't overlap causing undesired results (although I doubt you'd be using max-with much anyway but might be good to test and if it is an issue you'll know about it for any future projects)